creator/data_templates/field_definitions_schema.py

```diff
@@ -201,7 +201,9 @@ class FieldDefinitionsSchema(Schema):
         "changes": "Initial version"
     }
     key_order = list(FieldDefinitionSchema().fields.keys())
-
+    schema_version = fields.Dict(
```

Comment:
What is the difference between this and the `version` above?

Reply:
Maybe the naming is making it confusing. Perhaps it should be `_schema_version` and `schema_version`. The former contains the content of `schema_version` marshmallow field. 

In the `post_load` method I'm reading the version info and populating/adding a `schema_version` field in the deserialized data. If I don't declare the marshmallow field for `schema_version`, a validation error results bc `schema_version` would be an unknown field.